### PR TITLE
map pack testing

### PR DESCRIPTION
**Pull request by paologum**
added map pack tests

### CODE REVIEW

File: server/src/parse-map-packs.ts [hunk 1]
```diff
@@ -350,7 +259,158 @@ function grabRectBounds(headerLabels: { [key: number]: { label: string } }, info
...
+        vertBounds.centerLon = Number(info[i]);
+        break;
+      case "CENTER LAT":
+        vertBounds.centerLat = Number(info[i]);
+        break;
+      case "HEIGHT":
+        vertBounds.height = Number(info[i]);
+        break;
+      case "SCALE":
+        vertBounds.scale = Number(info[i]);
```
Comment (asivath) [inline server/src/parse-map-packs.ts:287]: everywhere you cast you should check for nan

Comment (paologum) [reply server/src/parse-map-packs.ts:287]: the fields of the variables are all checked to be the correct type in their corresponding `asserts`
these tests are just mainly for making sure the fields are being set
i mocked the asserts to all return true for isolation

File: server/src/parse-map-packs.ts [hunk 2]
```diff
@@ -350,7 +259,158 @@ function grabRectBounds(headerLabels: { [key: number]: { label: string } }, info
...
+    }
+  }
+  return vertBounds;
+}
+
+function grabTransects(headerLabels: string[], info: string[]) {
+  const transect: any = {};
+  let name = "";
+  for (let i = 1; i < info.length; i++) {
+    if (!info[i] || !headerLabels || !headerLabels[i] || !headerLabels[i]!) continue;
```
Comment (asivath) [inline server/src/parse-map-packs.ts:300]: !headerLabels[i]! shouldn't this cause an error?

File: server/src/parse-map-packs.ts (hunk 1 above)
Comment (asivath) [reply server/src/parse-map-packs.ts:287]: okay cool

File: server/src/parse-map-packs.ts (hunk 2 above)
Comment (paologum) [reply server/src/parse-map-packs.ts:300]: will take out

**Review by asivath — APPROVED** [review]
Looks good

**Pull request merged**